modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm

```diff
@@ -3,13 +3,13 @@
 	icon = 'modular_skyrat/modules/blueshield/icons/praetorian.dmi'
 	icon_state = "praetorian-control"
 	theme = /datum/mod_theme/blueshield
-	applied_cell = /obj/item/stock_parts/cell/super
+	applied_cell = /obj/item/stock_parts/cell/high // Bubberstation Edit - Original: Super Cell
 	applied_modules = list(
-		/obj/item/mod/module/storage/large_capacity,
```

Comment:
eeehhh they can affor d to keep it, they can buy the module anyway

Reply:
They can just buy it.